action.yml

```diff
@@ -4,16 +4,22 @@ branding:
   icon: 'package'
   color: 'blue'
 inputs:
-  environment:
-    description: 'Environment where is deployed'
+  environment: # required
+    description: 'Environment where your code is running. e.g. production'
     required: true
-  version:
+  version: # required
     description: 'Version deployed'
     required: true
-  status:
+  status: # optional
     description: 'Deploy status'
     required: false
     default: 'succeeded'
+  source_map: # optional. You can use upload-artifact and download-artifact actions.
```

Comment:
I think we can not do this with optional attributes as the `status` is optional as well we can not know if the `$3` is for `status` or for `source_map`.

Reply:
But don't we have default value for status? So even if someone keeps it empty it will still have some value.

Reply:
Oh right! I didn't realize that it will be always supplied when invoking the action! I thought we were defaulting only at API level, that we do as well. 

So, yes, then this shouldn't be a problem.